examples/Azure_IoT_Adu_ESP32/readme.md

```diff
@@ -0,0 +1,280 @@
+        Writing at 0x0000c000... (22 %)
+        Writing at 0x00010000... (27 %)
+        Writing at 0x00014000... (33 %)
+        Writing at 0x00018000... (38 %)
+        Writing at 0x0001c000... (44 %)
+        Writing at 0x00020000... (50 %)
+        Writing at 0x00024000... (55 %)
+        Writing at 0x00028000... (61 %)
+        Writing at 0x0002c000... (66 %)
+        Writing at 0x00030000... (72 %)
+        Writing at 0x00034000... (77 %)
+        Writing at 0x00038000... (83 %)
+        Writing at 0x0003c000... (88 %)
+        Writing at 0x00040000... (94 %)
+        Writing at 0x00044000... (100 %)
+        Wrote 396432 bytes (292339 compressed) at 0x00000000 in 13.0 seconds (effective 243.4 kbit/s)...
+        Hash of data verified.
+
+        Leaving...
+        Hard resetting via RTS pin...
+        ```
+
+        </p>
+        </details>
+
+2. Monitor the MCU (microcontroller) locally via the Serial Port.
+
+    - Go to menu `Tools`, `Serial Monitor`.
+
+        If you perform this step right away after uploading the sketch, the serial monitor will show an output similar to the following upon success:
+
+        ```text
+        Connecting to WIFI SSID buckaroo
+        .......................WiFi connected, IP address:
+        192.168.1.123
+        Setting time using SNTP..............................done!
+        Current time: Thu May 28 02:55:05 2020
+        Client ID: mydeviceid
+        Username: myiothub.azure-devices.net/mydeviceid/?api-version=2018-06-30&DeviceClientType=c%2F1.0.0
+        SharedAccessSignature sr=myiothub.azure-devices.net%2Fdevices%2Fmydeviceid&sig=placeholder-password&se=1590620105
+        MQTT connecting ... connected.
+        ```
+
+### Deploy Update
+
+To deploy the update to your ESP32, follow the link below:
+
+- [Deploy Update](https://docs.microsoft.com/azure/iot-hub-device-update/deploy-update)
+
+
+## Certificates - Important to know
+
+The Azure IoT service certificates presented during TLS negotiation shall be always validated, on the device, using the appropriate trusted root CA certificate(s).
+
+The Azure SDK for C Arduino library automatically installs the root certificate used in the United States regions, and adds it to the Arduino sketch project when the library is included.
+
+For other regions (and private cloud environments), please use the appropriate root CA certificate.
+
+### Additional Information
+
+For important information and additional guidance about certificates, please refer to [this blog post](https://techcommunity.microsoft.com/t5/internet-of-things/azure-iot-tls-changes-are-coming-and-why-you-should-care/ba-p/1658456) from the security team.
+
+## Troubleshooting
+
+- The error policy for the Embedded C SDK client library is documented [here](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/docs/iot/mqtt_state_machine.md#error-policy).
+- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-c/issues/new/choose).
```

Comment:
Should this point to the embedded c repo or the azure-sdk-for-c-arduino repo (or maybe both)? Same questions for the contributing section

Reply:
These are supposed to go to the `azure-sdk-for-c` since that's where the SDK lives.